plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -225,6 +225,60 @@ protected function blockQuoteContinue( $line, array $block ){
 	}
 
 	/**
+	 * Customize the inlineCode method
+	 *
+	 * @param array $block A block-level element
+	 * @access protected
+	 * @return string html representation generated from markdown.
+	 */
+	protected function inlineCode( $block ) {
+
+		$block = parent::inlineCode( $block );
+		
+		if( isset( $block['element']['text'] )) {
+			$block['element']['text'] = str_replace( '&amp;', '&', $block['element']['text'] );
```

Comment:
The logic there simply replaced any entity `&amp;` to `&`:

![screen shot 2017-02-25 at 9 01 09 am](https://cloud.githubusercontent.com/assets/955545/23326482/057808a8-fb39-11e6-8874-f531ad61d2c8.png)


1. if we enter `&` sign within the code block or backtick, the text processing will convert that into entity name, and that becomes `&amp;`.
2. Now, markdown will check if they found `&` sign, then it will converted to `&amp;`. So that the output string now becomes `<code>&amp;amp;</code>`

Let me know if that make sense.